articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -74,15 +74,26 @@ The following parameters are used by the App Configuration Push task:
         "TestApp:Settings:Message": "Message data"
     }
     ```
+- **File Content Profile**: The Configuration File's [content profile](https://docs.microsoft.com/azure/azure-app-configuration/concept-config-file). Default value is **Default**.
+     - **Default**: Refers to the conventional configuration file schema widely adopted.
```

Comment:
The "Default" profile doesn't refer to a specific schema, so this can be a bit confusing. How about something like this?

"_Refers to the conventional configuration file formats that are directly consumable by applications._"